cbits/filesystem.c

```
@@ -94,4 +94,100 @@ int dojang_file_type_at(int fd, const char *name)
     return 3;
 }
 
+/*
+ * Create one empty regular file relative to an already pinned directory.
+ * Return 1 on success or a negated errno.
+ */
+int dojang_create_empty_file_at(int fd, const char *name)
```

Cppcheck (2.21.0)

[style] The function 'dojang_create_empty_file_at' is never used.

(unusedFunction)